arch/arm64/boot/dts/rockchip/overlay/Makefile

```diff
@@ -118,7 +118,8 @@ dtbo-$(CONFIG_ARCH_ROCKCHIP) += \
 	rk3588-uart8-m0-full.dtbo \
 	rk3588-uart8-m1.dtbo \
 	rk3588-w1-gpio3-b3.dtbo \
-	rk3588-w1-gpio4-b1.dtbo
+	rk3588-w1-gpio4-b1.dtbo \
+	rk3588-ap6275p.dtbo
 
```

Comment:
What about to make it `orangepi-5-ap6275p` or `orangepi-5-wireless-ap6275p`

Reply:
That makes sense, I will rename the overlay to orangepi-5-ap6275p.